utils/wandb_utils.py

```diff
@@ -0,0 +1,16 @@
+import os
+from configs import wandb_config
+
+def wandb_watch(model, cfg: wandb_config):
+    import wandb
```

Comment:
As we're importing wandb only if we enable it we could potentially make it an optional dependency (compare vllm in pyproject.toml). Though I am not sure if its not installed as a dependency already.